React/creating-components.md

```diff
@@ -103,6 +103,28 @@ We used the existing index.js code to render the `App` component that was provid
 
 In the next phase of this application, we are going to extend our application to support multiple students and switch from hard-coded to dynamic data.
 
+### An example Student Component
```

Comment:
Thanks for doing all this work to create a more uniform flow with the student component!